graviti/portex/field.py

```diff
@@ -283,7 +227,7 @@ def rename(self, old_name: str, new_name: str) -> None:
         self._get_fields(old_name).rename(old_name, new_name)
 
 
-class Fields(NameOrderedDict[PortexType]):
+class Fields(NameOrderedDict[PortexType], FrozenFields):  # type: ignore[misc]
```

Comment:
where does the mypy misc comes from？

Reply:
Mypy considers that definition of "__setitem__" in base class "MutableMapping" is incompatible with the definition in base class "FrozenFields".